tealiumlibrary/src/main/java/com/tealium/core/settings/LibrarySettingsManager.kt

```diff
@@ -80,9 +82,9 @@ class LibrarySettingsManager(private val config: TealiumConfig,
                 cachedSettings
             }
             false -> {
-                loadFromAsset(assetString)?.also {
+                loadFromAsset(assetString).also {
+                    if (it != null) Logger.dev(BuildConfig.TAG, "Loaded local library settings.")
```

Comment:
Is this correct? If `it` is null then `isAssetSettingsLoaded` will still be true even though nothing was loaded - line below.